target/linux/ar71xx/image/generic.mk

```diff
@@ -229,6 +229,78 @@ define Device/dragino2
 endef
 TARGET_DEVICES += dragino2
 
+define Device/e1700ac-8M
+  DEVICE_TITLE := WHQX E1700AC-8M (8MB flash)
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage
```

Comment:
kmod-usb-storage is unnecessary, this device is not NAS. Better idea is add kmod-ath10k and ath10k-firmware-qca988x package

You define "usb led trigger" please add package "kmod-usb-ledtrig-usbport"